### PR TITLE
Issue #14937: Migrated section 4.1.1 Use of optional braces to follow chapter wise testing

### PR DESCRIPTION
**Pull request by Zopsss**
#14937

### CODE REVIEW

**Comment by Zopsss** [comment]
Failing CI: https://checkstyle.semaphoreci.com/jobs/1a06a3f8-0895-41a1-83b8-ca392ed90e5e

Not related to the PR

**Review by romani — APPROVED** [review]
Ok to merge

File: src/it/java/com/google/checkstyle/test/chapter4formatting/rule411optionalbracesusage/UseOfOptionalBracesTest.java [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+    @Override
+    protected String getPackageLocation() {
+        return "com/google/checkstyle/test/chapter4formatting/rule411optionalbracesusage";
+    }
+
+    @Test
+    public void testNeedBraces() throws Exception {
+        final String filePath = getPath("InputUseOfOptionalBraces.java");
+        verifyWithConfigParser(new String[] {"NeedBraces"}, filePath);
```
Comment (Zopsss) [inline src/it/java/com/google/checkstyle/test/chapter4formatting/rule411optionalbracesusage/UseOfOptionalBracesTest.java:36]: renamed the verify method in latest push

**Review by rdiachenko — APPROVED** [review]
lgtm

**Pull request merged**